web/themes/custom/votegov/templates/navigation/links--language-block.html.twig

```diff
@@ -64,7 +64,9 @@
         {%- if item.link -%}
           {# Replace default title with combo of endonym and english versions #}
           {% set link_title %}
-            <strong class="nonvfont">{{ item.link['#title'] }}</strong> {{ item.link['#english'] }}
+            <span{{ create_attribute().setAttribute('lang', key) }}>
```

Comment:
I think we can do the same thing with our the create attribute and set attribute and just hard code lang and use the key something like this.. what do you think?

I tested it out and seemed to work the same way.. 


```suggestion
            <span lang="{{ key }}">
```

Reply:
It does work the same way but I was keeping it consistent with other templates. Either works and is fine with me.